cluster/kubernetes/resource/spec.go

```diff
@@ -27,6 +27,11 @@ func (t PodTemplate) Containers() []resource.Container {
 		im, _ := image.ParseRef(c.Image)
 		result = append(result, resource.Container{Name: c.Name, Image: im})
 	}
+	for _, c := range t.Spec.InitContainers {
+		// FIXME(michael): account for possible errors here
```

Comment:
perhaps we can create an issue and write the issue number here?

Reply:
Good idea!